Source/WebKit/UIProcess/API/glib/WebKitSettings.cpp

```diff
@@ -3450,7 +3446,8 @@ gboolean webkit_settings_get_enable_media_stream(WebKitSettings* settings)
 {
     g_return_val_if_fail(WEBKIT_IS_SETTINGS(settings), FALSE);
 
-    return settings->priv->preferences->mediaStreamEnabled();
+    // FIXME: We may want to deprecate this and add new API for media devices.
+    return settings->priv->preferences->mediaDevicesEnabled();
```

Comment:
This change looks fine to me, but please create a bug report and include a link in this FIXME. Thanks!

Reply:
Updated!

Reply:
I would only deprecate the MediaStream settings, and allow changing MediaDevicesEnabled only using the feature flags API, though.